src/modules/service-catalog/components/useItemFormFields.tsx

```diff
@@ -90,12 +90,14 @@ export function useItemFormFields(
 
   useEffect(() => {
     const fetchAndSetFields = async () => {
```

Comment:
Same here: could be nice to extract this into a `useTicketFields` hook.

Reply:
Isn't it too much hooks? Because in this case useItemFormFields will only call another hook and pass fields and onChange method

Reply:
It's away to abstract the data loading part so the component can be a bit more simple and just hold the rendering logic. But it's fine to leave it here as well.